ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGenProducer.java

```diff
@@ -163,7 +164,7 @@ public void populateTopic(
     @Override
     public void onCompletion(final RecordMetadata metadata, final Exception e) {
       final String keyString = key == null ? "null" : key;
-      final String valueString = value == null ? "null" : value.toString();
+      final String valueString = value == null ? "null" : Objects.toString(value);
```

Comment:
The point of Objects.toString is that you don't have to do the null check.

Reply:
ah.. I didn't know that! Thanks, updated.